app/views/shared/_page_tabs.html.slim

```diff
@@ -27,16 +27,27 @@
       },
     }]
 
--if @work.supports_translation
+  -if @work.supports_translation
+    -tabs += [{\
+      :name => 'Translate',
+      :selected => 6,
+      :options => {\
+        :controller => 'transcribe',
+        :action => 'translate',
+      },
+    }]
+
+-if !current_user && !@work.restrict_scribes
   -tabs += [{\
-    :name => 'Translate',
-    :selected => 6,
+    :name => 'Transcribe',
+    :selected => 3,
     :options => {\
       :controller => 'transcribe',
-      :action => 'translate',
+      :action => 'guest',
     },
   }]
 
```

Comment:
The previous block doesn't quite make sense to me.  If no current_user and the work is not restricted:

formerly:  show translate
now:  show transcribe  ??

I suspect it's fine, but I need a bit of a walk-through because it doesn't "make sense" to me.